newIDE/app/src/MainFrame/index.js

```diff
@@ -3231,11 +3231,16 @@ const MainFrame = (props: Props) => {
                     onCloseProject: () => askToCloseProject(),
                     onOpenExampleStore: openExampleStoreDialog,
                     onSelectExampleShortHeader: onSelectExampleShortHeader,
-                    onPreviewPrivateGameTemplateListingData: onSelectPrivateGameTemplateListingData,
+                    onPreviewPrivateGameTemplateListingData: privateGameTemplateListingData =>
+                      onSelectPrivateGameTemplate({
+                        privateGameTemplateListingData,
+                        openDialog: true,
+                      }),
                     onOpenPrivateGameTemplateListingData: privateGameTemplateListingData => {
-                      onSelectPrivateGameTemplateListingData(
-                        privateGameTemplateListingData
-                      );
+                      onSelectPrivateGameTemplate({
+                        privateGameTemplateListingData,
+                        openDialog: false,
```

Comment:
nicely done.
Does it work when the NewProjectSetupDialog opens though?
Does it detect that there is a template selected and only shows the 3 fields? (and not the others)

Reply:
Yes, it works fine